source/versionInfo.py

```diff
@@ -31,11 +36,11 @@ def _updateVersionFromVCS():
 # Otherwise, py2exe will break.
 name="NVDA"
 longName=_("NonVisual Desktop Access")
-version="2016.3dev"
+version="%s.%s.%sdev"%(version_year,version_major,version_minor)

```

Comment:
This will result in, for example, 2016.3.0dev instead of just 2016.3dev. Perhaps this doesn't matter for now, as we override the version for releases anyway.